modular_skyrat/modules/novaya_ert/code/head.dm

```diff
@@ -0,0 +1,43 @@
+/obj/item/clothing/head/helmet/rus_helmet/nri
+	name = "Russian L47 helmet"
+	desc = "A Type 47 light helmet, it looks like it has a pocket for a bottle of vodka."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "russian_green_helmet"
+	mutant_variants = NONE
+	armor = list("melee" = 40, "bullet" = 35, "laser" = 15, "energy" = 30, "bomb" = 25, "bio" = 0, "rad" = 20, "fire" = 50, "acid" = 50, "wound" = 20)
+
+/obj/item/clothing/head/beret/sec/nri
+	name = "commander's beret"
+	desc = "Za rodinu!!"
+	armor = list("melee" = 40, "bullet" = 35, "laser" = 15, "energy" = 30, "bomb" = 25, "bio" = 0, "rad" = 20, "fire" = 50, "acid" = 50, "wound" = 20)
+
+/obj/item/clothing/head/helmet/nri_heavy
+	name = "altyn helmet"
+	desc = "A heavy Russian combat helmet with a strong ballistic visor. Alt+click to adjust."
+	icon_state = "russian_heavy_helmet"
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	armor = list(MELEE = 70, BULLET = 60, LASER = 50, ENERGY = 50, BOMB = 100, BIO = 100, RAD = 100, FIRE = 90, ACID = 90)
+	var/position = TRUE
```

Comment:
Would be nice to have a more significative name and some documentation on what it does.